src/App.jsx

```diff
@@ -65,15 +68,19 @@ const App = () => {
      * @function fetchData
      */
     async function fetchData() {
-      await generateUsersList(session);
-      await generateActivityTTL(session);
-      await updateUserActivity(session);
-      await createDocumentContainer(session);
+      setLoadingUsers(true);
+      setLoadingActive(true);
+      let podUrl = (await getPodUrlAll(session.info.webId, { fetch: session.fetch }))[0];
```

Comment:
Inrupt's `getPodUrlAll` function looks for pods declared on a user's profile in the `pim:storage` field. If there are no pods declared there (the field is not required) this function returns an empty array.

In that case we make an educated guess about where the pod should be.